packages/frontend/amp/components/AdConsent.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import { JsonScript } from './JsonScript';
+import { css, cx } from 'emotion';
+import { textSans } from '@guardian/pasteup/typography';
+import { palette } from '@guardian/pasteup/palette';
+import Tick from '@guardian/pasteup/icons/tick.svg';
+
+const consentUIStyle = css`
+    font-family: ${textSans(3)};
+    font-size: 16px;
+    line-height: 20px;
```

Comment:
We should be able to replace:

```css
    font-family: ${textSans(3)};
    font-size: 16px;
    line-height: 20px;
```

With just:

```js
${textSans(3)}
```

Or similar. Take a peek at the [Pasteup fonts docs](https://github.com/guardian/dotcom-rendering/tree/master/packages/pasteup#fonts) and let me know if you need a hand or the docs don't make sense.

Reply:
@SiAdcock Thanks Simon 👍 based on the doc I think `textSans(5)` is a closer option for the `font-size` and `line-height` so have changed this now.